regulations/static/regulations/css/scss/module/about.scss

```diff
@@ -46,37 +46,21 @@ hero
 
 .about-section {
   border-bottom: 1px solid $medium_gray;
-  padding-bottom: 1em;
-}
-
-.about-section:last-child {
-  border-bottom: 0px;
-}
-
-/*
-layout
--------------------
-*/
-
-
-.about-section {
-  padding: 4em 0;
+  padding: 4em 0 1em 0;
```

Comment:
There used to be two rules:
```css
padding-bottom: 1em;
// later
padding: 4em 0;
```

Wouldn't the first have been ignored? If so, this new rule isn't the same as what existed previously:
```
padding: 4em 0 1em 0;
```